### ISSUE TITLE
UI: epoch component

### ISSUE THREAD

**Issue by ryuash** [forbole/soljuno#83]
Does not seem like we keep a record of this with soljuno. Is this something I need to get from the rpc or will be logging this?
![image](https://user-images.githubusercontent.com/42913823/144363955-35990788-5ade-4638-899c-5742389ad3b0.png)

**Comment by dadamu** [comment]
One epoch is 432000 slots, let's see if it can be solved by a function, or by storing the info by a table.

**Comment by ryuash** [comment]
Oh i mean in this case, it's a fairly easy calculation if we are starting from slot 0. We can calculate this on the frontend

**Comment by dadamu** [comment]
@ryuash Okay, the epoch can be calculated with `slot / 432000`, then the time of until next epoch is `432000 - (slot%432000) * avg_slot_time`.

**Comment by ryuash** [comment]
Yes let me move this issue to my repo. No action is needed on your end except for getting the `avg_slot_time`

**Comment by dadamu** [comment]
@ryuash Okay, the avg_slot_time just have done by #84.
